internal/sirius/fee_string.go

```diff
@@ -0,0 +1,46 @@
+package sirius
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+)
+
+type FeeString string
```

Comment:
Is there a benefit to converting the amount to a string here? I feel like it would be easier to just treat it as an `int` until rendering it in the template

Reply:
Good point, its just made things more awkward. `int` it is!